### PR TITLE
doc(project): update readme.md for installation

### PR DESCRIPTION
**Pull request by anangkur**
updating readme for installation process

### CODE REVIEW

**Comment by mochadwi** [comment]
LGTM, looks like the author also use bintray and jitpack :D so it can use `jcenter()` and using lowercase of `image-support` instead of `Image-Support`. Beause jitpack using same naming with our github username & repo name

@anangkur

File: README.md [hunk 1]
```diff
@@ -5,15 +5,16 @@ Library to add ImageView (ImageBadgeView) with a badge like notification count.
...
+	repositories {
+		...
+		maven { url 'https://jitpack.io' }
+	}
 }
 ```
 Download via Gradle:
 ```
-implementation 'com.github.nikartm:image-support:1.0.5'
+implementation 'com.github.sampingantech:Image-Support:1.1.0'
```
Comment (adesamp) [inline README.md:17]: is it possible to make it lower-case like the origin? @anangkur

Comment (anangkur) [reply README.md:17]: actually I just copy and paste from jitpack. maybe the explanation from mas @mochadwi can explain this mas @adesamp 

> https://github.com/sampingantech/Image-Support/pull/2#issuecomment-757657498

**Comment by mochadwi** [comment]
> LGTM, looks like the author also use bintray and jitpack :D so it can use `jcenter()` and using lowercase of `image-support` instead of `Image-Support`. Beause jitpack using same naming with our github username & repo name
> 
> @anangkur 

Beause jitpack using same naming with our github username & repo name
, therefore we can't use lowercase naming 

Cmiiw 

But let's try them both (lowercase vs. capitalize) in our Sampingan apps, wdyt? 

@anangkur @adesamp

File: README.md (hunk 1 above)
Comment (mochadwi) [reply README.md:17]: action items? https://github.com/sampingantech/Image-Support/pull/2#issuecomment-757674931

**Pull request merged**